integration-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Thoth-Station Integration Tests
+
```

Comment:
copy the topic definition from the readme: https://github.com/thoth-station/integration-tests
and paste it here and also add the link.